src/components/input/DatePickerWithLabelFormGroup.tsx

```diff
@@ -22,6 +23,7 @@ const DatePickerWithLabelFormGroup = (props: Props) => {
         dateFormat="MM/dd/yyyy"
         dateFormatCalendar="LLLL yyyy"
         dropdownMode="scroll"
+        maxDate={addDays(new Date(), 0)}
```

Comment:
This component is intended to be more general than for use inside of the new/edit patient screen. 

I think that this interface will only need to expose an optional maxDate and just pass it along to the `DateTimePicker from `@hopsitalrun/components`. 

Then we'll need to make a change in the `GenrealInformationComponent` to set the `maxDate` property.

Reply:
I agree 100% with your vision. @jlato89 can you handle it?

Reply:
I should be able to yes. just an oversight on my part.